### DISCUSSION TITLE
both ROOK_EXTERNAL_ADMIN_SECRET and ROOK_EXTERNAL_USER_SECRET is not supported, choose one only

### DISCUSSION THREAD

**Discussion by dborgi** [rook/rook#12820] [Q&A]
Hello how are you ? I am trying to connect my okd version 4.13 cluster with my external ceph cluster and when I run the bash script import-external-cluster.sh I get the error Providing both ROOK_EXTERNAL_ADMIN_SECRET and ROOK_EXTERNAL_USER_SECRET is not supported, choose one only.

Could you please help me ? thank you

**Comment by subhamkrai** [comment]
@parth-gr ^^
